### PR TITLE
Contact page

### PR DESCRIPTION
**Pull request by dandelionadia**

### CODE REVIEW

File: src/pages/components/TitleBar.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { IoIosArrowForward } from 'react-icons/io'
+import styled from 'styled-components'
+import { Box } from 'atomic-layout'
+
+const StyledTitleBar = styled.div`
+  background-color: ${({ theme }) => theme.colors.greyDim};
+  background-image: url('https://demo2.wpopal.com/auros/wp-content/uploads/2018/10/breadcrumb.jpg');
```
Comment (kettanaito) [inline src/pages/components/TitleBar.tsx:9]: Maybe consider hosting this image yourself? Just to be safe if that Auros resource is not available, so your website is not broken.

File: src/pages/contactUs/atoms/Label.tsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import styled from 'styled-components'
+
+export const Label = styled.label`
+  color: ${({ theme }) => theme.colors.blackLight};
+  font-size: 16px;
+  font-weight: 600;
+  line-height: 28px;
```
Comment (kettanaito) [inline src/pages/contactUs/atoms/Label.tsx:7]: You can also use plain numbers (i.e. `1.5`) that would be calculated from the current element's `font-size`:

```css
font-size: 16px;
line-height: 1.5; // 1.5 * 16px = 24px
```

File: src/pages/contactUs/components/Form.tsx [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+  height: 120px;
+`
+
+const StyledButton = styled(Button)`
+  background-color: ${({ theme }) => theme.colors.blackLight};
+  border: 2px solid ${({ theme }) => theme.colors.blackLight};
+  font-size: 14px;
+  margin: 30px 0;
+  padding: 18px 28px;
+  max-width: max-content;
```
Comment (kettanaito) [inline src/pages/contactUs/components/Form.tsx:34]: What does this do?

File: src/pages/contactUs/components/TitleBlock.tsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  font-weight: 600;
+  padding-top: 5px;
+  text-transform: uppercase;
+`
+
+interface PropsTitleBlock {
+  title: string
+}
+
+export const TitleBlock: React.FC<PropsTitleBlock> = ({ title }) => {
```
Comment (kettanaito) [inline src/pages/contactUs/components/TitleBlock.tsx:22]: What is the difference between `TitleBlock` and `TitleBar` components?

File: src/pages/components/TitleBar.tsx (hunk 1 above)
Comment (dandelionadia) [reply src/pages/components/TitleBar.tsx:9]: I will do it in the future if you think necessary. But now all data depend on auros data.

File: src/pages/contactUs/components/Form.tsx (hunk 3 above)
Comment (dandelionadia) [reply src/pages/contactUs/components/Form.tsx:34]: I've reused the button and change some of the properties

File: src/pages/contactUs/components/TitleBlock.tsx (hunk 4 above)
Comment (dandelionadia) [reply src/pages/contactUs/components/TitleBlock.tsx:22]: Annotation 2020-06-15 113607

File: src/pages/contactUs/atoms/Label.tsx (hunk 2 above)
Comment (dandelionadia) [reply src/pages/contactUs/atoms/Label.tsx:7]: is my option wrong?

Comment (kettanaito) [reply src/pages/contactUs/atoms/Label.tsx:7]: Nope, just a suggestion :)

File: src/molecules/Header.tsx [hunk 5]
```diff
@@ -7,6 +7,7 @@ import { NavMenu } from './NavMenu'
 import { Logo } from '../atoms/Logo'
 import { Bar } from './Bar'
 import routes from '../routes'
```
Comment (kettanaito) [inline src/molecules/Header.tsx:9]: I don't think you use this `routes` import in this file.

Comment (dandelionadia) [reply src/molecules/Header.tsx:9]: I used it here :
```
<Link to={routes.home}>
```

File: src/pages/components/TitleBar.tsx [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+`
+
+const StyledSpan = styled.span`
+  color: ${({ theme }) => theme.colors.black};
+  font-size: 13px;
+  padding-left: 10px;
+`
+
+interface propsTitleBar {
+  titlePage: string
```
Comment (kettanaito) [inline src/pages/components/TitleBar.tsx:39]: What do you think about naming it just `title: string`?

File: src/pages/contactUs/components/Form.tsx [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+    event.preventDefault()
+
+    const values = {
+      name: nameValue,
+      email: emailValue,
+      message: messageValue,
+      subscribed,
+    }
+
+    console.warn(values)
```
Comment (kettanaito) [inline src/pages/contactUs/components/Form.tsx:86]: Please remove this `console.warn` call. Thanks!

File: src/pages/contactUs/components/Form.tsx [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+  align-items: center;
+`
+
+export const Form: React.FC = () => {
+  const [nameValue, setNameValue] = useState('')
+  const [emailValue, setEmailValue] = useState('')
+  const [messageValue, setMessageValue] = useState('')
+  const [subscribed, setSubscribed] = useState(false)
+
+  const handleNameChange = (event: any) => {
```
Comment (kettanaito) [inline src/pages/contactUs/components/Form.tsx:60]: Maybe let's add proper TypeScript types here instead of `any`? We can discuss this.

**Comment by kettanaito** [comment]
Please resolve the build errors and feel free to merge :)

**Pull request merged**